ts/Extensions/Boost/BoostSeries.ts

```diff
@@ -937,11 +937,17 @@ function scatterProcessData(
     // Filter unsorted scatter data for ranges
     const processedData: Array<PointOptions> = [],
         processedXData: Array<number> = [],
-        processedYData: Array<number> = [];
+        processedYData: Array<number> = [],
+        xRangeNeeded = !(isNumber(xExtremes.max) || isNumber(xExtremes.min)),
+        yRangeNeeded = !(isNumber(yExtremes.max) || isNumber(yExtremes.min));
```

Comment:
I think we should look for `..Extremes.dataMax` and `dataMin` instead of the derived `min` and `max`

Reply:
I had issues with zoom in/out and boost enter/exit not working as expected.